salt/base/config/sources.list.jinja

```diff
@@ -4,12 +4,13 @@
 {% set base_repo = "us.archive.ubuntu.com/ubuntu/" %}
 {% endif %}
 
+{# TODO: Investigate, these seem to be handled doubly in sources.list.d as well as sources.list
```

Comment:
Investigate, these seem to be handled doubly in sources.list.d as well as sources.list

Reply:
We also need to ensure back compat. with previous OS releases